wiki/osu!_Program_Files/en.md

```diff
@@ -1,113 +1,115 @@
+## Hidden folders
 
-#### Data
+These folders are hidden because any modifications to them could prevent osu! from starting correctly, or at all.
 
-**osu!data files.** Should not be tampered.
+### Data
 
-## Program files
+osu! data files. Contains some of osu!'s cache, like beatmap background cache and avatar caches. They should not be deleted, because they may be in use by osu!.
 
-### .db (Database file)
+## Files
 
-The .db files are beatmaps' data which only osu!.exe/osu!test.exe can utilise. Notepad can open the .db files but it will be mostly made up of encrypted characters.
+*Caution: Be careful with these files, you might break osu! if you are not careful.*
 
-**osu! database**
+## Database files (.db)
 
-- collection.db (Your "Collections" in-game. You can post it to forum for others to check out your "Collections". Actual beatmaps not provided.)
-- osu!.db (osu! beatmaps' cache. You know, that long start-up loading screen before you see Song Selection?)
-- presence.db (Cache of osu!players @ Chat Console)
-- scores.db (Stores historical Local scores)
+The database files are daabases that osu! requires to function propertly. The files contain vital information that osu! requires, such as saved scores, and the list of beatmaps saved on the player's device.
 
-### .cfg (Configuration files)
+- `collections.db`: Stores the player's "Collections" in-game. The player can post them to the forums for others to see. Actual beatmaps are not provided with the collection file.
+- `osu!.db`: Stores osu!'s database of beatmaps.
+- `presence.db`: Stores a cache of osu!players logged in the Chat Console.
+- `scores.db`: Stores the local leaderboards.
 
-Configuration files or config files configure the initial settings for osu! to work. The files can be opened by Notepad.
+## .cfg (Configuration files)
+
+Configuration files or config files configure the initial settings for osu! to work. The files can be opened with a text editor.
 
 - `osu!.cfg`: Stores security information about the osu! application files and current release stream. This should never be modified manually.
-- `osu!.<your PC account name>.cfg`: Stores [Options](/wiki/Options) data and other game settings. See [User Configuration File](/wiki/osu!_Program_Files/User_Configuration_File).
+- `osu!.<operating system username>.cfg`: Stores [Options](/wiki/Options) data and other game settings. See [User Configuration File](/wiki/osu!_Program_Files/User_Configuration_File).
 
 ### .exe (Application)
 
-The main component. Click on it to start-up. The .exe files are safe to open assuming you used the osu!installer to install osu!.
+The main component. Click on it to start-up (only applies to Windows.) The .exe files are safe to open assuming the player used the osu!installer to install osu!.
 
 osu!.exe (Start-up osu!)
 
-### Hidden files
-
 #### .dll (application extension)
 
-These .dll files are components of osu!.
+These .dll files are osu!'s components and dependencies.
```

Comment:
Not sure of the value of including this. It's already outdated.

Reply:
I think maybe the list should go, but the text that describes what they're for should stay.

Reply:
Done in 57ca705